### PR TITLE
cabal: clean-up of dep bounds

### PR DESCRIPTION
**Pull request by Anton-Latukha**
Clean-up.

### CODE REVIEW

File: hnix.cabal [hunk 1]
```diff
@@ -940,31 +940,31 @@ executable hnix
...
-    , bytestring
-    , comonad
-    , containers
-    , data-fix
-    , deepseq
-    , exceptions
-    , filepath
-    , free
+      hnix
+    , aeson >= 1.4.2 && < 1.6
```
Comment (mightybyte) [inline hnix.cabal]: You don't have to duplicate bounds that already exist elsewhere in the cabal file.  You only need to put bounds for the dependencies here that don't have bounds elsewhere.

**Review by mightybyte — CHANGES_REQUESTED** [review]

File: hnix.cabal (hunk 1 above)
Comment (Anton-Latukha) [reply hnix.cabal]: Thanks for curation.

Did `cabal gen-bounds` for executable and then added `library` override.

Would redo.

Well, then would also check/remove other bound copies.

Comment (Anton-Latukha) [reply hnix.cabal]: Thanks, indeed - bounds were normal, additionally made bounds unique.

**Review by mightybyte — APPROVED** [review]

**Review by sjakobi — APPROVED** [review]
Thanks! :)

**Pull request merged**